### PR TITLE
fix pallet-xcm toml related to runtime-benchmarks

### PR DESCRIPTION
**Pull request by girazoki**
I tried compiling cumulus from master and I failed. I think the errors is caused by how runtime-benchmarks features is declared in pallet-xcm. This fix makes it compile again

### CODE REVIEW

File: xcm/pallet-xcm/Cargo.toml [hunk 1]
```diff
@@ -42,5 +42,5 @@ std = [
 ]
 runtime-benchmarks = [
 	"frame-system/runtime-benchmarks",
-	"xcm-builder/runtime-benchmarks",
+	"xcm-builder",
```
Comment (bkchr) [inline xcm/pallet-xcm/Cargo.toml:45]:
```suggestion
```

This is the proper solution.

File: xcm/pallet-xcm/Cargo.toml [hunk 2]
```diff
@@ -18,12 +18,12 @@ frame-system = { git = "https://github.com/paritytech/substrate", default-featur
 
 xcm = { path = "..", default-features = false }
 xcm-executor = { path = "../xcm-executor", default-features = false }
+xcm-builder = { path = "../xcm-builder", default-features = false, optional = true }
```
[the remark this answers is not in the document]
Comment (girazoki) [reply xcm/pallet-xcm/Cargo.toml:21]: before this PR it was declared as to being importable when compiling with runtime-benchmarks. AFAIK is runtime-benchmarks feature flag can be added without doing cargo tests no?

File: xcm/pallet-xcm/Cargo.toml (hunk 1 above)
Comment (girazoki) [reply xcm/pallet-xcm/Cargo.toml:45]: I see. so its not needed for benchmarks as well

**Comment by girazoki** [comment]
Updated @bkchr

**Review by bkchr — APPROVED** [review]

**Review by ordian — APPROVED** [review]

**Comment by ggwpez** [comment]
@shawntabrizi would your TOML linter have found this?

**Review by KiChjang — APPROVED** [review]

**Pull request merged**